java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -198,6 +199,13 @@ public enum UserDefinedInstanceProperty implements InstanceProperty {
     DEFAULT_COMPACTION_FILES_BATCH_SIZE("sleeper.default.compaction.files.batch.size", "11"),
     DEFAULT_SIZERATIO_COMPACTION_STRATEGY_RATIO("sleeper.default.table.compaction.strategy.sizeratio.ratio", "3"),
     DEFAULT_SIZERATIO_COMPACTION_STRATEGY_MAX_CONCURRENT_JOBS_PER_PARTITION("sleeper.default.table.compaction.strategy.sizeratio.max.concurrent.jobs.per.partition", "" + Integer.MAX_VALUE),
+    COMPACTION_EC2_TYPE("sleeper.compaction.ec2.type", "t3.xlarge", Utils::isNonNullNonEmptyString),
+    COMPACTION_EC2_POOL_MINIMUM("sleeper.compaction.ec2.pool.minimum", "0", Utils::isNonNegativeInteger),
+    COMPACTION_EC2_POOL_DESIRED("sleeper.compaction.ec2.pool.desired", "0", Utils::isNonNegativeInteger),
+    COMPACTION_EC2_POOL_MAXIMUM("sleeper.compaction.ec2.pool.maximum", "3", Utils::isNonNegativeInteger),
```

Comment:
Given that we set the maximum number of concurrent Fargate tasks to 300 by default, should this be > 3?

Reply:
30?